lib/Cake/Model/Behavior/TranslateBehavior.php

```diff
@@ -74,9 +93,16 @@ public function setup(Model $Model, $config = array()) {
 			return false;
 		}
 
-		$this->settings[$Model->alias] = array();
+		$this->settings[$Model->alias] = array(
+			'fields' => array(),
+			'join_type' => 'INNER',
```

Comment:
Changing the structure of public properties will break compatibility for someone. Is there another way to store this that doesn't require potentially breaking user land code?